client/my-sites/earn/ads/payments.jsx

```diff
@@ -1,41 +1,22 @@
 import { Badge, Card } from '@automattic/components';
 import classNames from 'classnames';
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
 import QueryWordadsPayments from 'calypso/components/data/query-wordads-payments';
 import QueryWordadsSettings from 'calypso/components/data/query-wordads-settings';
 import Notice from 'calypso/components/notice';
 import { CALYPSO_CONTACT } from 'calypso/lib/url/support';
+import { useSelector } from 'calypso/state';
 import { getWordadsSettings } from 'calypso/state/selectors/get-wordads-settings';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import { getWordAdsPayments } from 'calypso/state/wordads/payments/selectors';
 
-class WordAdsPayments extends Component {
-	static propTypes = {
-		siteId: PropTypes.number,
-		payments: PropTypes.arrayOf(
-			PropTypes.shape( {
-				id: PropTypes.number.isRequired,
-				paymentDate: PropTypes.string,
-				amount: PropTypes.string,
-				status: PropTypes.string,
-				paypalEmail: PropTypes.string.isRequired,
-				description: PropTypes.string,
-			} )
-		).isRequired,
-	};
+const WordAdsPayments = ( { numberFormat } ) => {
+	const translate = useTranslate();
```

Comment:
BTW we have core version available, and I've been wondering if we should also switch to it while rewriting things.

```js
import { useI18n } from '@wordpress/react-i18n';
const { __ } = useI18n();
```

Not needed now, but just food for thought for future. We can check with team global.